pyslackers_website/marketing/views.py

```diff
@@ -27,3 +27,11 @@ def get_context_data(self, **kwargs):
 
 class TermsOfService(TemplateView):
     template_name = 'marketing/terms_of_service.html'
+
+
+def info(request):
+    app_version = os.getenv("app_version")
+    if app_version:
+        return HttpResponse(app_version)
```

Comment:
Let's return a JSON Response so it's easier to consume. Instead of the check, just show an empty string. The JSON body should look like:

```
{
    "revision": "<sha here or empty string>"
}
```

And you can remove the `if app_version` and other message if it's not found (and empty string does that)